dask/tests/test_distributed.py

```diff
@@ -398,3 +335,93 @@ async def test_annotation_pack_unpack(c, s, a, b):
     unpacked_hlg = HighLevelGraph.__dask_distributed_unpack__(packed_hlg, annotations)
     annotations = unpacked_hlg["annotations"]
     assert annotations == {"workers": {"n": ("alice",)}}
+
+
+class SchedulerImportCheck(SchedulerPlugin):
```

Comment:
What do you think about moving this to a new `test_layers.py` module?

Reply:
Good call - done.